src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -0,0 +1,28 @@
+module FileConventions.Test
+
+open System
+open System.IO
+
+open NUnit.Framework
+open NUnit.Framework.Constraints
+
+open FileConventions
+
+[<SetUp>]
+let Setup () =
+    ()
+
+[<Test>]
+let MixedLineEndingsTest1 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithMixedLineEndings")))
+    Assert.That(MixedLineEndings fileInfo, Is.EqualTo true)
+
+[<Test>]
+let MixedLineEndingsTest2 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithoutMixedLineEndings")))
+    Assert.That(MixedLineEndings fileInfo, Is.EqualTo false)
+
+[<Test>]
+let MixedLineEndingsTest3 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithCRLFLineEndings")))

```

Comment:
@realmarv please save `Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles"` in a DirectoryInfo variable to not have to repeat it in every test; this is a bad practice whose fix is usually called "DRY" (Don't Repeat Yourself), and I'd expect you from now on detect this duplication yourself (when you're about to commit, in the `git gui` view, it's very easy to spot duplication by looking at your own diff)

Reply:
I removed all duplicates but I'm not sure if was ok to remove duplicates of other tests as well